src/middlewared/middlewared/plugins/chart_releases_linux/upgrade.py

```diff
@@ -156,6 +162,33 @@ async def upgrade(self, job, release_name, options):
 
         return chart_release
 
+    @private
+    def upgrade_values(self, release, new_version_path):
```

Comment:
I know we talked about a migration using python, but i would if its possible to make it language agnostic, passing the config as a temporary file location in json format?

Thoughts?